### PR TITLE
Enable automatic login after password reset

### PR DESCRIPTION
**Pull request by sigberto**

### CODE REVIEW

**Comment by sigberto** [comment]
@stvstnfrd

File: common/djangoapps/student/views.py [hunk 1]
```diff
@@ -2329,6 +2329,9 @@ def password_reset_confirm_wrapper(
                 entry = PasswordHistory()
                 entry.create(updated_user)
 
+            updated_user.backend = 'django.contrib.auth.backends.ModelBackend'
```
Comment (stvstnfrd) [inline common/djangoapps/student/views.py]: Why do we need this?

Comment (sigberto) [inline common/djangoapps/student/views.py]: honestly, not sure. i took it from the superuser_login_as method. it doesn't work otherwise.

**Comment by stvstnfrd** [comment]
retest this please

File: common/djangoapps/student/views.py (hunk 1 above)
Comment (stvstnfrd) [inline common/djangoapps/student/views.py]: Cool.

**Comment by stvstnfrd** [comment]
+1

**Comment by stvstnfrd** [comment]
@kluo Can you put eyes on this too?

**Comment by stvstnfrd** [comment]
@kluo You good on this?
Hopefully we can include this in the release.

**Comment by caesar2164** [comment]
i remember this, 👍

**Pull request merged**